site/content/installation/ngf-images/pulling-ngf-image.md

```diff
@@ -0,0 +1,104 @@
+- An NGINX Gateway Fabric subscription. Download both the certificate (*nginx-repo.crt*) and key (*nginx-repo.key*) from the [MyF5 Customer Portal](https://my.f5.com). Keep in mind that an NGINX Plus certificate and key won't work for for the steps in this guide.
+
+
+## Set up Docker for F5 Container Registry
+
+Start by setting up Docker to communicate with the F5 Container Registry located at `private-registry.nginx.com`. If you're using Linux, follow these steps to create a directory and add your certificate and key:
+
+```shell
+mkdir -p /etc/docker/certs.d/private-registry.nginx.com
+cp <path-to-your-nginx-repo.crt> /etc/docker/certs.d/private-registry.nginx.com/client.cert
+cp <path-to-your-nginx-repo.key> /etc/docker/certs.d/private-registry.nginx.com/client.key
+```
+
+The steps provided are for Linux. For Mac or Windows, consult the [Docker for Mac](https://docs.docker.com/desktop/faqs/general/#how-do-i-add-custom-ca-certificates) or [Docker for Windows](https://docs.docker.com/desktop/faqs/windowsfaqs/#how-do-i-add-custom-ca-certificates) documentation. For more details on Docker Engine security, you can refer to the [Docker Engine Security documentation](https://docs.docker.com/engine/security/).
+
+
+## Pull the image
+
+Next, pull the image you need from `private-registry.nginx.com`. To find the correct image, consult the [Technical Specifications](https://github.com/nginxinc/nginx-gateway-fabric#technical-specifications).
+
+To pull an image, follow these steps. Replace `<version-tag>` with the specific version you need, for example, `1.2.0`.
+
+- For NGINX Plus Gateway Fabric, run:
+
+  ```shell
+  docker pull private-registry.nginx.com/nginx-gateway-fabric/nginx-plus:<version-tag>
+  ```
+
+You can use the Docker registry API to list the available image tags by running the following commands. Replace `<path-to-client.key>` with the location of your client key and `<path-to-client.cert>` with the location of your client certificate. The `jq` command is used to format the JSON output for easier reading.
+
+```json
+$ curl https://private-registry.nginx.com/nginx-gateway-fabric/nginx-plus/tags/list --key <path-to-client.key> --cert <path-to-client.cert> | jq
+{
+  "name": "nginx-gateway-fabric/nginx-plus",
+  "tags": [
+    "edge"
+  ]
+}
+```
+
+
+## Push to your private registry
+
+After pulling the image, tag it and upload it to your private registry.
+
+1. Log in to your private registry:
+
+   ```shell
+   docker login <my-docker-registry>
+   ```
+
+1. Tag and push the image. Replace `<my-docker-registry>` with your registry's path and `<version-tag>` with the version you're using, for example `1.2.0`:
+
+   - For NGINX Plus Gateway Fabric, run:
+
+      ```shell
+      docker tag private-registry.nginx.com/nginx-gateway-fabric/nginx-plus:<version-tag> <my-docker-registry>/nginx-gateway-fabric/nginx-plus:<version-tag>
+      docker push <my-docker-registry>/nginx-gateway-fabric/nginx-plus:<version-tag>
+      ```
+
+
```

Comment:
```suggestion
1. Tag the image, replacing `<my-docker-registry>` with your registry's path and `<version-tag>` with the version you're using:

    ```shell
        docker tag private-registry.nginx.com/nginx-gateway-fabric/nginx-plus:<version-tag> <my-docker-registry>/nginx-gateway-fabric/nginx-plus:<version-tag>
   
1. Push the image:

    ```shell
        docker push <my-docker-registry>/nginx-gateway-fabric/nginx-plus:<version-tag>
    ```
```